awx/api/templates/instance_install_bundle/install_receptor.yml

```diff
@@ -15,4 +15,8 @@
       pip:
         name: ansible-runner
         executable: pip3.9
+    - name: Install receptorctl
+      pip:
+        name: receptorctl
```

Comment:
This will be slightly faster if you do it as part of the previous task and pass a list instead of a string (which gets turned into a list anyway internally).

Something like:
```yaml
- name: Install dependencies from pip
  pip:
    name:
      - ansible-runner
      - receptorctl
    executable: pip3.9
```